generator/app.go

```diff
@@ -686,10 +776,16 @@ func writeYaml(data interface{}, path string) error {
 }
 
 func main() {
+	var err error
+	releases, err = getLastThreeK8sReleases()
```

Comment:
An improvement to this would be if the generator takes the releases as an optional argument and defaults to the last 3 releases.

This way we can control which releases need to be part of the section if needed.

Reply:
Valid improvement, but also can make a TODO